cmd/dm-worker/main_test.go

```diff
@@ -56,6 +56,7 @@ func TestRunMain(_ *testing.T) {
 	defer func() { utils.OsExit = oldOsExit }()
 	utils.OsExit = func(code int) {
 		log.L().Info("os exits", zap.Int("exit code", code))
+		_ = utils.KillMySelf()
```

Comment:
I can't remember why we need this helper function now. maybe the process won't exit after this test function returns?

if so, I think the reason may be that lthe test function should be named `TestMain` not `TestRunMain`, please check if this works.